TestBenches/InputRouter_test.cpp

```diff
@@ -398,14 +417,38 @@ void prepareInputStreams( ifstream * pInputStreams
   }
 }
 
+// for now .. create LUT for IR 
+// this can eventually be produced 
+// by either the wiring script 
+// or the emulation 
+void prepareLUTs( int pDTCsplit = 0
+  , int pNonant = 4 
+  , std::string pInputFile_Wires = "emData/LUTs/wires.dat" )
+{
+  auto cDtcMap = getCablingMap( pInputFile_Wires );
+  auto cMapIter = cDtcMap.begin(); 
+  uint8_t cLinkId=0;
+  do
+  {
+    auto cDtcName = cMapIter->first;
+    auto cLyrs = cMapIter->second; 
+    auto cNmems = getNMemories( cLinkId, pDTCsplit, pNonant, pInputFile_Wires);
+    auto cBnWrd = cNmems.second; 
+    cLinkId++;  
+    cMapIter++;
+  }while( cMapIter != cDtcMap.end() );
+}
```

Comment:
@sseifeln The only thing this function seems to do is create IR_Nmems.tab via the call to `getNMemories`. And then I don't see IR_Nmems.tab being used before it is removed at the end. I also don't see how `cDtcName`, `cLyrs`, `cNmems`, and `cBnWrd` are used here. Can this be cleaned up?

Reply:
Sorry.. legacy! Now removed.